apps/nextra/pages/en/build/smart-contracts/debugging.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: "Debugging Move"
+---
+
+# Debugging Move
+
+Move was designed to be simple and safe, but like with all programming languages,
+bugs can still occur. This guide will help you debug your Move code and figure out
+what went wrong.
+
+Please feel free to contribute with additional tooling and information that can
+help others in the community.
+
+## Debugging with the Aptos CLI
+
+### Simulation on transaciton submission
+
+You can use the Aptos CLI to simulate entry functions prior to executing them.
+
+Normally, a transaction will fail in simulation if it won't work on-chain. For example:
+
+```bash name="Terminal"
+aptos move run --function-id 0x1::aptos_account::transfer --args address:0x1 u64:1000000000000000000                                     λ:main  [    ]
```

Comment:
I think the `λ:main [    ]` at the end was not intended? I see this in a few places on this page.

Reply:
woops, copied from my CLI